ush/rrfs_radmon/exnam_verfrad.sh

```diff
@@ -0,0 +1,205 @@
+#/bin/bash
+
+# Run data extract/validation for regional radiance diag data
+echo "---> exnam_vrfyrad.sh.ecf"
+
+#  Command line arguments
+export PDY=${1:-${PDY:?}} 
+export cyc=${2:-${cyc:?}}
+
+#################################################################################
+# Set the required Directories by x.zhang
+#################################################################################
+
+#export Homeradmon=/scratch2/NCEPDEV/fv3-cam/Xiaoyan.Zhang/save/plot/RadMon_rrfs/GSI-Monitor
+#export Homenam=$Homeradmon/src/Radiance_Monitor/nwprod
+export FIXnam=${FIXnam:-${FIX_GSI}}
+
+#################################################################################
+
+#export USHnam=${USHnam:-${HOMEnam}/ush}
+export USHnam=${USHnam:-${USHDIR}/rrfs_radmon}
+#export USHradmon=${USHradmon:-$HOMEradmon/ush}
+export USHradmon=${USHradmon:-${USHnam}}
```

Comment:
If I understand correctly, USHnam = USHradmon. If so, please replace "USHnam" with "USHradmon" in the scripts. I have a question. In the NCO standards, we typically use "USH{model}". Is "radmon" a model name? @MatthewPyle-NOAA, what do you think about this?

Reply:
@chan-hoo radmon is a util package name

Reply:
should I change USHradmon to USHrrfs for everywhere, and delete USHnam?

Reply:
I think "USHradmon=USHrrfs/rrfs_radmon", so if USHradmon is ok, you won't have to do this. However, USHnam=USHradmon. So it would be better to remove USHnam and replace it with USHradmon.

Reply:
replaced all USHnam by USHradmon .